src/addrman.h

```diff
@@ -318,8 +329,15 @@ friend class CAddrManTest;
     {
         LOCK(cs);
 
-        unsigned char nVersion = 2;
-        s << nVersion;
+        const auto stream_version_orig = s.GetVersion();
+
+        if (m_format >= Format::V3) {
```

Comment:
AddrMan already had versions on its on-disk database with numbers. So I left the convention as is and just added new version "3" since the last one used was "2". This has nothing to do with "addrv2" or "torv3" :)

`IsAddrV2Format()` would spoil the strict ordering of the AddrMan on-disk versions.

Reply:
Yes, I understand. Helper functions that replace the various `m_format >= Format::V<n>` may possibly be nice, but opinions can vary and it isn't a blocker.